src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -476,64 +506,35 @@ const PIAFormPage = () => {
     if (!pia?.title) {
       invalid = true;
       formId = 'title';
+      setValidationMessages((prevState) => ({
+        ...prevState,
+        piaTitle: 'Error: Please enter a title.',
+      }));
     }
-    if (!pia?.ministry && !invalid) {
+    if (!pia?.ministry) {
       invalid = true;
       formId = 'ministry-select';
+      setValidationMessages((prevState) => ({
+        ...prevState,
+        piaMinistry: 'Error: Please select a ministry.',
+      }));
     }
-    if (!pia?.branch && !invalid) {
+    if (!pia?.branch) {
       invalid = true;
       formId = 'branch';
+      setValidationMessages((prevState) => ({
+        ...prevState,
+        piaBranch: 'Error: Please enter a branch.',
+      }));
     }
-    if (!pia?.drafterName && !invalid) {
-      invalid = true;
-      formId = 'drafterName';
-    }
-    if (!pia?.drafterTitle && !invalid) {
-      invalid = true;
-      formId = 'drafterTitle';
-    }
-    if ((!pia?.drafterEmail || pia.drafterEmail === undefined) && !invalid) {
-      invalid = true;
-      formId = 'drafterEmail';
-    }
-    if (!pia?.leadName && !invalid) {
-      invalid = true;
-      formId = 'leadName';
-    }
-    if (!pia?.leadTitle && !invalid) {
-      invalid = true;
-      formId = 'leadTitle';
-    }
-    if (!pia?.leadEmail && !invalid) {
-      invalid = true;
-      formId = 'leadEmail';
-    }
-    if (!pia?.mpoName && !invalid) {
-      invalid = true;
-      formId = 'mpoName';
-    }
-    if (!pia?.mpoEmail && !invalid) {
-      invalid = true;
-      formId = 'mpoEmail';
-    }
-    if (!pia?.initiativeDescription && !invalid) {
+
+    if (!pia?.initiativeDescription) {
       invalid = true;
       formId = 'initiativeDescription';
-    }
-    if (!pia?.initiativeScope && !invalid) {
-      invalid = true;
-      formId = 'initiativeScope';
-    }
-    if (!pia?.dataElementsInvolved && !invalid) {
```

Comment:
I do believe this should still be validated. As by default we need either a 'yes' or a 'no' or 'null'. But there can be an instance where non of the elements are checked (someone trying to right click go to inspect element etc .. and send nothing checked) the api will give an error. The frontend should not allow this to happen.  MY 2 cents.

Reply:
agree, add back